controllers/articles.controllers.js

```diff
@@ -22,3 +22,13 @@ exports.getAllArticles = (req, res, next) => {
       next(err);
     });
 };
+
+exports.updateArticle = (req, res, next) => {
+  const id = req.params.article_id
+  const {inc_votes} = req.body
+  articlesModel.modifyArticle(id, inc_votes).then((article) =>{
```

Comment:
I would destructure each model when you require them in rather than using them like this. It would be more readable that way 👍